### DISCUSSION TITLE
Android packages are ignored when using manifest mode

### DISCUSSION THREAD

**Discussion by AleksandrsJakovlevs80lvl** [microsoft/vcpkg#24247] [Q&A] [answered]
I was trying to install android libraries using vcpkg manifest, but they are simply skipped for some reason. I am lost, pls tell me what I am doing wrong.

This is my manifest:
```
{
    "name": "native-deps",
    "version": "1.0.0",
    "dependencies": [
        {
            "name": "openssl",        
            "version>=": "1.1.1n",
            "platform": "(x86 & android) | (x86 & windows)"
        }		
    ],                        
    "overrides": [
        { "name": "openssl", "version-string": "1.1.1n" },
        { "name": "abseil", "version-string": "20211102.1" }
    ],
    "builtin-baseline": "6e024e744e7717c06ddacd5089401109c6298553"
}
```

I am running this command:
`vcpkg\vcpkg.exe --feature-flags="versions" install "--clean-after-build" "--x-install-root=.\vcpkg\installed" "--binarysource=clear;interactive;nuget,VCPkg"`

This is the output:
[log.txt](https://github.com/microsoft/vcpkg/files/8512801/log.txt)

These are the resulting folders:
![image](https://user-images.githubusercontent.com/57263826/164000053-c1acceca-65ee-42e0-8ba0-c0c6ac26d8ff.png)

It works if I do `vcpkg.exe install openssl:x86-android` - I can see the x86-android in installed folder - but I'd like to do it via manifest mode.

What am I missing?

**Comment by autoantwort** [comment, marked as answer]
You have to specify the target triplet, the default one is `x86-windows`. So you have to run
`vcpkg\vcpkg.exe install --triplet=x86-android "--clean-after-build" "--x-install-root=.\vcpkg\installed" "--binarysource=clear;interactive;nuget,VCPkg"`

You don't need `"platform": "(x86 & android) | (x86 & windows)"`. It means "I only depend on this library if I build for `(x86 & android) | (x86 & windows)`". So if you select x64-windows as your target triplet, it won't be installed.
